crates/ruff_python_formatter/tests/snapshots/black_compatibility@miscellaneous__string_quotes.py.snap

```diff
@@ -82,13 +82,12 @@ f"\"{a}\"{'hello' * b}\"{c}\""
 +f"NOT_YET_IMPLEMENTED_ExprJoinedStr"
 +f"NOT_YET_IMPLEMENTED_ExprJoinedStr"
  r"raw string ftw"
--r"Date d\'expiration:(.*)"
-+r"Date d'expiration:(.*)"
+ r"Date d\'expiration:(.*)"
  r'Tricky "quote'
 -r"Not-so-tricky \"quote"
 -rf"{yay}"
 -"\nThe \"quick\"\nbrown fox\njumps over\nthe 'lazy' dog.\n"
-+r'Not-so-tricky "quote'
++r'Not-so-tricky \"quote'
```

Comment:
There seems to be a related issue that we don't count the `"` in `preferred_quotes` for raw strings. Do you want to take a look at this as well, and if so, as part of this PR or do you prefer a second pr?

Reply:
@MichaReiser 

> There seems to be a related issue that we don't count the " in preferred_quotes for raw strings.

Can you elaborate on this?

Reply:
Anyway, I prefer a second PR.